__tests__/app.test.js

```diff
@@ -158,20 +158,52 @@ describe("8. GET /api/articles", () => {
           expect(article).toHaveProperty("votes");
           expect(article).toHaveProperty("comment_count");
           expect(article).toHaveProperty("author");
-          // expect(article).toEqual(
+        });
+      });
+  });
+});
 
-          //   // expect.objectContaining({
-          //   //   title: expect.any(String),
-          //   //   article_id: expect.any(Number),
-          //   //   topic: expect.any(String),
-          //   //   created_at: expect.any(String),
-          //   //   votes: expect.any(Number),
-          //   //   comment_count: expect.any(Number),
-          //   // })
-          // );
+describe("9. GET /api/articles/:article_id/comments", () => {
+  test("returns an array of comments for given article id", () => {
+    const article_ID = 9;
+    return request(app)
+      .get(`/api/articles/${article_ID}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toBe(2);
+        expect(comments).toBeInstanceOf(Array);
+        comments.forEach((comment) => {
+          expect(comment).toBeInstanceOf(Object);
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
         });
```

Comment:
you would also want to check that each comment has the article_id that you requested for